include/stl2/detail/range/concepts.hpp

```diff
@@ -159,6 +159,20 @@ STL2_OPEN_NAMESPACE {
 	STL2_CONCEPT ViewableRange =
 		Range<Rng> &&
 		(_RangeImpl<Rng> || View<Rng>);
+
+	namespace ext {
+		template<Range R>
```

Comment:
Design question: Why constrain these with `Range` instead of constraining `iterator_t` and `sentinel_t`?

Reply:
Per personal discussion on Slack, @cjdb is going to add this suggestion to P1035.

Reply:
https://www.cjdb.com.au/wg21/ranges/d1035/d1035.html#introducing-associated-types-for-ranges

Reply:
Reflected here: https://www.cjdb.com.au/wg21/ranges/d1035/d1035.html#amendment-to-the-associated-iterator-types

Reply:
I didn't realize that the Better Safer CPO paper made this possible. Very nice! (y)